src/vscode-dts/vscode.proposed.chatSessionsProvider.d.ts

```diff
@@ -138,6 +137,22 @@ declare module 'vscode' {
 		// TODO: Should we introduce our own type for `ChatRequestHandler` since not all field apply to chat sessions?
 		// TODO: Revisit this to align with code.
 		readonly requestHandler: ChatRequestHandler | undefined;
+
+		/**
+		 * Options for the session
+		 */
+		readonly options?: {
```

Comment:
Maybe `capabilities`? And should this be on the provider instead if it's going to be the same for all sessions from it? Something like:

```ts
registerChatSessionContentProvider(type, provider, capabilities?: { supportsHotReload? boolean })
```

Reply:
Nice, I like `capabilities`.